### PR TITLE
Update user stories, target user profile and value proposition

### LINKED ISSUE #36: Update DG user stories

### PR DESCRIPTION
**Pull request by github-amanda**
Close #36

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -236,29 +236,28 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
+* residents in a household
 * has a need to manage a significant number of contacts
```
Comment (kumsssss) [inline docs/DeveloperGuide.md]: could change contacts to household items

**Review by kumsssss — CHANGES_REQUESTED** [review]
just a minor change. other than that, LGTM.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (github-amanda) [reply docs/DeveloperGuide.md]: ok

**Review by kumsssss — APPROVED** [review]
LGTM

**Pull request merged**